tools/taskcluster/decision_task.py

```diff
@@ -0,0 +1,66 @@
+import argparse
```

Comment:
Are these files similar to the Fenix ones? If so, I wonder if we should be sharing the files with some kind of dependency manager instead of duplicating them across repos (though it does add an additional point of failure so maybe it's not worth it).